src/main/java/teammates/common/datatransfer/attributes/InstructorAttributes.java

```diff
@@ -142,12 +144,12 @@ public String getIdentificationString() {
 
     @Override
     public String getEntityTypeAsString() {
-        return "Instructor";
+        return DEFAULT_DISPLAY_NAME;
     }
 
     @Override
     public String getBackupIdentifier() {
-        return Const.SystemParams.COURSE_BACKUP_LOG_MSG + courseId;
+        return INSTRUCTOR_BACKUP_LOG_MSG + courseId + email;
```

Comment:
There should be a delimiter between courseId and email.

Reply:
Is it better to use the delimiter as a variable or simply append a "::" after courseId?